src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditYear.razor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditYear : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public int? Year { get; set; }
+
+    [Parameter]
+    public string? Title { get; set; }
+
+    [Parameter]
+    public EventCallback<int?> YearChanged { get; set; }
+
+    public string ErrorRequiredMessage
+    {
+        get
+        {
+            return Title + " is required";
```

Comment:
@DanielHodgeGH  we need to discuss localization for this string that are shown to the user

can y take note of it?

Reply:
have created https://regusit.atlassian.net/browse/NOX-573